components/Styled-Components/AdultNext.js

```diff
@@ -12,7 +12,9 @@ const AdultNext = styled.button`
   margin-left: auto;
   margin-right: auto;
   margin-top: 10rem;
-  width: 13rem;
+  position: absolute;
+  bottom: 0rem;
```

Comment:
The adult next button is now appearing in the bottom left, behind the back button. I don't know if you noticed this